database/redshift/redshift.go

```diff
@@ -64,16 +65,16 @@ func WithInstance(instance *sql.DB, config *Config) (database.Driver, error) {
 
 	config.DatabaseName = databaseName
 
-	if len(config.MigrationsTable) == 0 {
```

Comment:
default config value fallback removed

Use original ordering for a cleaner diff. Also, config values should be "set" before any instances or connections are made.